.eslintrc

```diff
@@ -7,5 +7,8 @@
         "ecmaVersion": 8,
         "sourceType": "module",
         "ecmaFeatures": {}
+    },
+    "rules": {
+      "no-throw-literal": 0
```

Comment:
touche, and I'm surprised that it's suggesting "0" and not "off".  https://github.com/builtforme/swatchjs/pull/26